client/deposits/declarations.d.ts

```diff
@@ -91,4 +91,6 @@ declare module '@woocommerce/components' {
 	}
 
 	const TourKit: ( props: any ) => JSX.Element;
+
+	const List: ( props: { className: string; items: any[] } ) => JSX.Element;
```

Comment:
Let's change this to be:

```suggestion
	interface ListProps {
		className?: string;
		items: any[];
	}
	const List: React.FC< ListProps >;
```

to be more consistent with the currently recommended TS guidelines 👍

Reply:
Fixed in [fb30723](https://github.com/Automattic/woocommerce-payments/pull/6249/commits/fb307234c017411253d6e67db59c183c33c3b7b2).